sdk/component/component.go

```diff
@@ -20,22 +20,31 @@ type (
 		Read(ctx context.Context) ([]*ocsf.VulnerabilityFinding, error)
```

Comment:
Simplified by taking care of the storage out of the box.

Components are not required to implement Read, Write, Update or Close no more.